### PR TITLE
Feature/cluster size

### PR DESCRIPTION
**Pull request by waynedovey**
Bash Menu-Driven options for:

- OpenShift Version
- OpenShift Cluster Size 
- OpenShift Disconnected setting 
- Enable OpenShift Container Storage (OCS)

### CODE REVIEW

File: cluster-build-gsslab.sh [hunk 1]
```diff
@@ -1,33 +1,79 @@
 #!/bin/bash
 
 DEFAULT_OCPVERSION=4.3.8
+CLUSTER_SIZE=small
+WORKER_MEMORY=8192
+WORKER_CPU=2
+DISCONNECTED=false
+OCS_SETTING=false
+BUILD_LAB=gsslab
```
Comment (luckylittle) [inline cluster-build-gsslab.sh]: We should have one build script then and not `cluster-build-gsslab` etc.

File: openshift.ver [hunk 2]
```diff
@@ -1 +1 @@
-4.3.8
\ No newline at end of file
+small
```
Comment (luckylittle) [inline openshift.ver]: I don't get this? `small`?

File: vars/vars.yml.template [hunk 3]
```diff
@@ -2,14 +2,15 @@
 bootstrap:
   ignHttpLocation: http://192.168.0.x:8080/ignition/bootstrap.ign
 set_proxy: false
-disconnected: false
+disconnected: {{ disconnected_setting }}
```
Comment (luckylittle) [inline vars/vars.yml.template:5]: Let's add all the possible variables to README

File: cluster-build-gsslab.sh (hunk 1 above)
Comment (waynedovey) [reply cluster-build-gsslab.sh]: I agree, that's the next task

File: openshift.ver (hunk 2 above)
Comment (waynedovey) [reply openshift.ver]: Fixed, thanks

File: vars/vars.yml.template (hunk 3 above)
Comment (waynedovey) [reply vars/vars.yml.template:5]: Cool, I will update

**Review by luckylittle — APPROVED** [review]
Well done

**Pull request merged**